core/Debian/stretch/Dockerfile

```diff
@@ -21,3 +22,6 @@ RUN apt-get update \
     && mkdir -p /etc/ansible \
     && echo "[local]" > /etc/ansible/hosts \
     && echo "localhost ansible_connection=local" >> /etc/ansible/hosts
+
+COPY davedaemon /usr/local/bin/
+RUN chmod 755 /usr/local/bin/davedaemon
```

Comment:
Hi,

Using travis-ci you currently bootup the docker image in background and expect it to run on a long term to then execute the tests using docker exec ... commands.

So you need a long term process to run this image without consuming all the cpu of the machine. This was traditionnaly done using the init process which just sleep for something to happen once it has started what needed. which is basically nothing inside a docker image.

However to save disk space and remove unused stuff by default, because an init system inside a docker image does not really make sense, systemd is not installed in debian stretch docker images now.

We can simply reinstall it in this image but it seems a little bit too much useless stuff for me. So I use this small dave fake daemon which is just a sleep loop.
I've tried with base binaries but didn't find something that works. (tried /bin/true, cat, ...) but they fail to wait for input because there is no tty to attach to. If you know another base binary which can just sleep for something to happen to keep the docker container running in background davedaemon is useless.

Thanks.